tests/test_model_label_checker.py

```diff
@@ -0,0 +1,52 @@
+"""Tests for model label construction checker."""
```

Comment:
I gave it a try and I see the following issues with the approach:

- just a single input / error file for each checker, if I understand the code correctly
    - when I'm checking multiple inputs, some of them should be fine, some not, this can be non-deterministic
    - not an easy way to run specific parameters only, must rewrite the input file
- dependent on pylint formatting
- testing single feature is spread to multiple files, worse to get oriented
- pytest fails when running locally
- pytest pass even when the `error_SOMETHING.txt` is empty, try: ```
echo > tests/input/error_nautobot_incorrect_base_class.txt && invoke build && invoke pytest 
...
=========================================================================================
9 passed, 1 warning in 3.22s ==========================================================================================
```

Don't want to pull different direction :-), just pls. consider my notes, overall it seems to me to violate least surprise principle. I can go your way and fix some mentioned issues or refactor current tests to use pylint's `assertAddsMessages` approach. WDYT?